sklearn/utils/_weight_vector.pyx.tp

```diff
@@ -1,6 +1,27 @@
+{{py:
+
+"""
+Efficient (dense) parameter vector implementation for linear models. 
+
+Template file for easily generate fused types consistent code using Tempita
+(https://github.com/cython/cython/blob/master/Cython/Tempita/_tempita.py).
+
+Generated file: weight_vector.pxd
+
+Each class is duplicated for all dtypes (float and double). The keywords
+between double braces are substituted in setup.py.
+"""
+
+# name, c_type
+dtypes = [('64', 'double'),
+          ('32', 'float')]
+
+}}
+
 # cython: cdivision=True
 # cython: boundscheck=False
 # cython: wraparound=False
+# cython: language_level=3
```

Comment:
the language level is set globally here https://github.com/scikit-learn/scikit-learn/blob/45bb9ab916351c95a6d15a5cb06469036a0707cf/sklearn/_build_utils/__init__.py#L40